origin-growth/migrations/20190124003426-create-growth-event.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const { GrowthEventTypes } = require('../src/enums')
+
+const tableName = 'growth_event';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      type: {
+        type: Sequelize.ENUM(GrowthEventTypes)
+      },
+      eth_address: {
```

Comment:
In the [spec](https://docs.google.com/document/d/1QN9xf-4jIEdMyTyOZc2ocJHAjPCgSNwFGAVdL4ZfyZM/edit?ts=5c480f25#) this table also has `status` enum: 
```
status: enum [Logged, Verified, Fraud]
```

Reply:
Good catch. Added.